base/ca/shared/conf/CS.cfg

```diff
@@ -818,6 +818,7 @@ internaldb._002=##
 internaldb.basedn=
 internaldb.maxConns=15
 internaldb.minConns=3
+internaldb.maxResults=1000
```

Comment:
I think we don't need this since the default will be set programmatically. We just need to document it somewhere to make sure people can configure it if they need it.

Reply:
I think `CS.cfg` serves as documentation as well. This way new installs will get it (and it is documented). The code ensures we don't need a migration. Also, where should we document it?

Reply:
As discussed on IRC, let's start a man page on CS.cfg params like this one:
https://github.com/dogtagpki/pki/blob/master/docs/manuals/man5/pki-tps-connector.5.md
This can be done in a separate PR.

Also as discussed we want to set the default maxResults to 0 (i.e. unset), so I don't think we need to add anything in CS.cfg.